lms/djangoapps/instructor/enrollment.py

```diff
@@ -244,6 +245,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
                     requesting_user_id=requesting_user_id
                 )
                 submission_cleared = True
+                SCORE_CHANGED.send(
```

Comment:
I put it here so I would have access to `block`, but getting `block` can be moved so that this can go at the end of the function.